src/client/Microsoft.Identity.Client/Internal/Broker/IBroker.cs

```diff
@@ -13,5 +13,7 @@ internal interface IBroker
         bool CanInvokeBroker();
 
         Task<MsalTokenResponse> AcquireTokenUsingBrokerAsync(Dictionary<string, string> brokerPayload);
+
+        Task<IEnumerable<IAccount>> GetAccountsAsync(string clientID);
```

Comment:
Please add a comment stating that is only supported in conjuction with BrokerSupportsSilentFlow flag